lib/infra/quests/db.ex

```diff
@@ -23,7 +23,7 @@ defmodule Infra.Quests.Db do
     {:ok, new_quest}
   end
 
-  def write(quest, %Event.AdventurerJoinedParty{} = event) do
+  def write(quest, event) do
```

Comment:
Other than the create event that needs to start up the db server I don't think the events need to be handled differently from the infra perspective. They will always project the change and then update the cache.